source/scss/components/_quicksearch.scss

```diff
@@ -22,6 +22,14 @@
   }
 }
 
+.qs-with-background.inline {
+  margin-bottom: -70px;
```

Comment:
Would be possible to fix the element position so don't use negative margins?

Reply:
maybe in a future ticket (we have some upcoming tickets about background image, etc)